### PR TITLE
Web Accessibility

### PR DESCRIPTION
**Pull request by ramiaquista**
### Accessibility Changes :  🔨

- Added change to Page Title in order to be more specific
- Added images alternative text to people who can't see the img.
- Text Headings -  No issue found.
- Color Contrast - No issue found.
- Resize Text breakpoint fixed.
- Interaction - No issue found.
- Moving content - No issue found.
- Multimedia - No issue found.
- The basic structure of the page - No issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Ramiro Aquistapace</title>
+    <title>Ramiro's Portfolio</title>
```
Comment (ramiaquista) [inline index.html:7]: Added a more specific title to be more specific and clear!

File: style.css [hunk 2]
```diff
@@ -479,7 +479,7 @@ form textarea {
   color: white;
 }
 
-@media only screen and (min-width: 992px) {
+@media only screen and (min-width: 1100px) {
```
Comment (ramiaquista) [inline style.css:482]: Added new min-width in order to fix a text resize breaking point.

File: index.html [hunk 3]
```diff
@@ -59,36 +59,36 @@ <h2 class="connect">LET’S CONNECT</h2>
...
               <a href="#"
-                ><img src="img/Tw-Icon.png" alt="" class="dflex i-btn"
+                ><img src="img/Tw-Icon.png" alt="Twitter Icon" class="dflex i-btn"
               /></a>
             </li>
             <li>
               <a
                 href="https://www.linkedin.com/in/ramiro-aquistapace-32b61b204/"
-                ><img src="img/Linkedin-Icon.png" alt="" class="dflex i-btn"
+                ><img src="img/Linkedin-Icon.png" alt="LinkedIn Icon" class="dflex i-btn"
```
Comment (ramiaquista) [inline index.html:68]: Added alternative text to images for people who can't see the images.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @ramiaquista 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it.
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**